code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -27,6 +27,8 @@
 	matter = list("metal" = 150)
 	origin_tech = "materials=1;engineering=1"
 	attack_verb = list("bashed", "battered", "bludgeoned", "whacked")
+	tool_behaviour = TOOL_WRENCH
+	toolspeed = 1
```

Comment:
This is `1` by default already. No need to redefine it.

Reply:
Whoops fixing now. I will leave a commented message at the top of this file saying that toolspeed affects how fast it works.